### DISCUSSION TITLE
Changing order of columns in Experiment screens

### DISCUSSION THREAD

**Discussion by arees88** [mlflow/mlflow#7085] [Q&A] [answered]
Hi,

**Re:  Changing order of columns in Experiment screens**

Is it still possible to change the order of Parameters and Metrics columns in Mlflow experiments?

In earlier versions of Mlflow there were two dialog boxes:

**Filter Params 
Filter Metrics**

They allowed to select the list of desired columns + preserve the order in the list.

The Filter lists were easily copied from one experiment to another making it very convenient to have consistent view across the experiments.

I can see there is **Columns** dialog that allows to select columns, but it does not seem to allow to change column order.

Is it possible to have the old-style **Filter Params** and **Filter Metrics** available as well? I did use this feature a lot.

Many thanks,

A Rees

**Comment by arees88** [comment]
F.Y.I.
I enclose a couple of screen shots showing the old-style **Filter Params** examples:

**Example 1 – `Exec` param as 1st column**

![Filter_Params_Example_1](https://user-images.githubusercontent.com/60096624/196472994-43200530-2f0a-4774-afa6-b229b4edc350.png)

**Example 2 – `Exec` param as last column**

![Filter_Params_Example_2](https://user-images.githubusercontent.com/60096624/196472995-dd6dee70-9e1e-47df-b3fd-df0f9cd58385.png)

**Comment by JHibbard** [comment, marked as answer]
Hi A Rees,

Thanks for highlighting this behavior. I never noticed that the filter param order was preserved in the resulting table and definitely never considered it as a way to provide consistent views across experiments. Would you be willing to file this as feature request, referencing this discussion?

Thank you,
Jim

**Reply by arees88** [reply]
Hi Jim,

Many thanks for your prompt reply and feedback. 

I have created FR #7112. 

Regards,
A Rees
